### DISCUSSION TITLE
Consider showing UI elements/tabs in live preview / HUD modals

### DISCUSSION THREAD

**Discussion by wsydney76** [craftcms/cms#8896] [Ideas]
### Description

Great feature to hide a title when the context is obvious by tab or heading, however this context is gone when using live preview;  the input is orphaned, the user lost.

In general we would like to (optionally?) see tab names and ui elements also in preview, as it can be a long unstructured form, difficult to orientate yourself, and missing useful tips and warnings.

**Comment by brandonkelly** [comment]
Related: #6587

**Comment by mmikkel** [comment]
Really hope to see this at some point soon-ish. 

IMO the big problem with Preview has always been that the author loses all edit form context and focus when it opens. Having tabs and UI elements inside the Preview panel would be super helpful in this regard (especially if Craft was able to retain the currently active tab and/or focused field from the edit form), but the ideal solution would be if there was a way to open the preview panel without actually replacing the actual edit form.

**Comment by brandonkelly** [comment]
This is halfway resolved for 3.7 – we’ve solved it for element editors, but not Live Preview yet. See #8017.

**Reply by Wiejeben** [reply]
Would be awesome to see the slideout like tabs in the live preview sidebar!

**Reply by thupsi** [reply]
+1 from me! On large/wide screens especialy I see no reason to lose tabs and UI elements

**Reply by development-lameco** [reply]
+1, Especially when working with multiple Tabs it would be usefull to at least show those tab names as deviders.

**Reply by brandonkelly** [reply]
Live Preview is gaining full support for tabs, UI elements, and conditions, in Craft 5.2 🎉 (#15112)

**Reply by brandonkelly** [reply]
Craft 5.2.0-beta.1 is out with those changes.

**Comment by MoritzLost** [comment]
+1000 for showing tabs and UI elements in the live preview. Some complex element forms _really_ need some structure provided by tabs, dividing lines and headings. Furthermore, some fields require some explanations beyond field instructions that can be provided through the _Tip_ or _Template_ UI elements. Those forms are borderline unusable in the live preview, where all tabs and UI elements are gone.
